config/config.exs

```diff
@@ -29,6 +29,29 @@ config :plug, :statuses, %{
   460 => "Invalid device token"
 }
 
+lager_formater_config = [:date, 'T', :time, :color, ' [', :severity, '] ', :pid, ' ', :message, '\e[0m\r\n']
+config :lager,
+  colored: true,
+  handlers: [
+    lager_console_backend: [
+      level: :info,
+      formatter: :lager_default_formatter,
+      formatter_config: lager_formater_config
+    ],
+    lager_file_backend: [
```

Comment:
Why the split between two log files? Does it mimic the default lager behaviour?

Reply:
Yes, the goal here was to copy what `lager` is doing by default.